include/amici/model_ode.h

```diff
@@ -160,7 +164,48 @@ namespace amici {
         virtual void fJSparse(SUNMatrixContent_Sparse JSparse, const realtype t,
                               const realtype *x, const realtype *p,
                               const realtype *k, const realtype *h,
-                              const realtype *w, const realtype *dwdx) = 0;
+                              const realtype *w, const realtype *dwdx) {
+            throw std::invalid_argument("Not implement for this model!");
+            // don't use AmiException as the backtrace causes some hefty
```

Comment:
Is that really a concern when an non-recoverable exception is thrown?

Will this exception pass through runAmiciSimulation? Need to catch std::exception separately and print, not via catchall which only prints generic message.